evergreen/system_perf/6.0/genny_tasks.yml

```diff
@@ -837,17 +835,6 @@ tasks:
       test_control: transport_layer_connect_timing
   name: transport_layer_connect_timing
   priority: 5
-- commands:
-  - command: timeout.update
-    params:
-      exec_timeout_secs: 86400
-      timeout_secs: 86400
-  - func: f_run_dsi_workload
-    vars:
-      auto_workload_path: src/genny/src/workloads/query/AggregateExpressions.yml
```

Comment:
[question] Are only the regular 'AggregateExpressions' causing false positives, unstable results, etc? That is, do the [VariadicAggregateExpressions](https://github.com/mongodb/genny/blob/7401c77020d0911ffbeb8c77182353416ef89ee2/src/workloads/query/VariadicAggregateExpressions.yml) not cause the same issues?

Reply:
Please see this [chart](https://charts.mongodb.com/charts-performance-data-mbycr/dashboards/60aba70e-8375-4b91-88da-ef168f9a696f?view=ddcd216b-b96c-48df-81e7-998a5c855f09) to have an idea of which workload produces how many false positive change points.
aggregate_expressions is a true leader in that sense, with coll_scan_.* following it